modules/pel/peltool/peltool.py

```diff
@@ -541,11 +541,30 @@ def printPELCount(path: str, config: Config, extension: str):
     print("{\n    \"Number of PELs found\": "+str(count)+"\n}")
 
 
+class CustomFormatter(argparse.HelpFormatter):
```

Comment:
Nice, but I'm surprised that the standard argparse doesn't support a list.

Could you please point some reference on this?

Reply:
Sure, please refer [link1](https://github.com/jtamames/SqueezeMeta/blob/master/lib/checkm/customHelpFormatter.py#L28), [link2](https://stackoverflow.com/questions/9642692/argparse-help-without-duplicate-allcaps).